apps/web/src/components/AccessRisk/index.tsx

```diff
@@ -52,16 +52,24 @@ export const TOKEN_RISK_T = {
   [TOKEN_RISK.UNKNOWN]: <Trans>Unknown Risk</Trans>,
 } as const
 
-function RetryRisk({ onClick }: { onClick: () => void }) {
+function RetryRisk({ onClick, isFetching }: { onClick: () => void; isFetching: boolean }) {
```

Comment:
Would it be better to accept a async call instead of managing the `isFetching` state? For example
```typescript
type RetryRiskProps = {
  // Rename from onClick to onRetry, and return a promise
  onRetry: () => Promise<void>
}
```

then inside the component we will know whether the call ends or not. Further more we can know if the call is successful or not